pkg/util/indexeres/pod.go

```diff
@@ -0,0 +1,20 @@
+package indexeres
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/harvester/harvester/pkg/ref"
+	"github.com/harvester/harvester/pkg/util"
+)
+
+const (
+	PodByVMNameIndex = "harvesterhci.io/pod-by-vmname"
+)
+
+func PodByVMName(obj *corev1.Pod) ([]string, error) {
+	vmName, ok := obj.Labels[util.LabelVMName]
+	if !ok {
+		return []string{}, nil
```

Comment:
Feel free to ignore suggestion, but I prefer nil as it doesn't heap allocate.

```suggestion
		return nil, nil
```